x-pack/plugins/enterprise_search/server/routes/enterprise_search/search_applications.ts

```diff
@@ -64,7 +71,9 @@ export function registerSearchApplicationsRoutes({ log, router }: RouteDependenc
       const engine = (await client.asCurrentUser.searchApplication.get({
```

Comment:
Maybe we could start changing `engine` for `search_app` when we change code related to search apps like this